requirements-tests.txt

```
@@ -5,8 +5,8 @@ pytest-cov >=2.10.0,<8.0.0
 coverage[toml] >=6.2,<8.0
 pytest-xdist >=1.32.0,<4.0.0
 pytest-sugar >=0.9.4,<1.2.0
-mypy ==1.4.1
-ruff ==0.12.12
+mypy ==1.18.1
+ruff ==0.13.0
```

These are unrelated edits, let's remove them for simplicity (bots are pushing updates of these dependencies in separate PRs)
```suggestion
mypy ==1.4.1
ruff ==0.12.12
```